.travis.yml

```diff
@@ -1,30 +1,50 @@
-sudo : true
-dist : trusty
+os: linux
 language: node_js
-node_js:
-  - "node"
+node_js: "14"
+
 addons:
-  sauce_connect: true
   ssh_known_hosts: download.kiwix.org
-before_script:
-    - npm install nightwatch http-server
-    - "./node_modules/http-server/bin/http-server . &"
-script:
-    # The free account on Sauce does not allow more than 5 concurrent sessions (including the main one)
-    # So we separate the recent and old browsers in order to respect this limit.
-    # NB : latest edge should be added to this list when Microsoft releases the new Edge based on Chromium
-    - "./node_modules/nightwatch/bin/nightwatch -c nightwatch.js --env firefox,chrome"
-    - "./node_modules/nightwatch/bin/nightwatch -c nightwatch.js --env edge40,edge44"
-    - "./node_modules/nightwatch/bin/nightwatch -c nightwatch.js --env firefox45,chrome58,ie11"
-    - pkill node || echo "Node process not running (anymore)"
-deploy:
-    - provider: script
-      # Nightly builds (launched by cron)
-      script: sudo apt-get update -q && sudo apt-get --yes install click click-reviewers-tools && ./scripts/setup_travis_env.sh && DISPLAY=:99.0 ./scripts/create_all_packages.sh
-      on:
-          condition: ( "$TRAVIS_EVENT_TYPE" = "cron" )
-    - provider: script
-      # Generation of packages for public releases (launched by a tag)
-      script: sudo apt-get update -q && sudo apt-get --yes install click click-reviewers-tools && ./scripts/setup_travis_env.sh && DISPLAY=:99.0 ./scripts/create_all_packages.sh -t -v ${TRAVIS_TAG}
-      on:
-          tags: true
+
+jobs:
+  fast_finish: true
+  include:
+
+    - stage: "Local testing"
+      node_js: "10"
+      addons:
+        firefox: latest-esr
+        chrome: stable
+      script: npm test
```

Comment:
There should be a comment here to explain why there is this stage.
At first sight, it seems already covered by the npm run `test-unit-saucelabs` in next stage (which does it on even more browser types)

Reply:
It is here so that when that job is skipped (like for PRs) that we still run it locally with Firefox and Chrome within the CI system, and also so that what we recommend running locally for development (`npm test`) actually works which is important to verify even for local branches and PRs so that it doesn't regress for some reason, since it does run slightly differently from SauceLabs.

I've added a comment to that effect.